.github/workflows/build.yml

```diff
@@ -27,25 +27,15 @@ jobs:
     - name: Run NUKE
       run: ./build.ps1
       env:
-        NuGetApiKey: ${{ secrets.NUGETAPIKEY }}
-
-    - name: Check for 'lcov.info' existence
-      id: check_files
-      uses: andstor/file-existence-action@v2
-      with:
-        files: "TestResults/reports/lcov.info"
-
-    - name: coveralls
-      uses: coverallsapp/github-action@1.1.3
-      if: steps.check_files.outputs.files_exists == 'true'
-      with:
-        github-token: ${{ secrets.GITHUB_TOKEN }}
-        path-to-lcov: TestResults/reports/lcov.info
+        ApiKey: ${{ secrets.NUGETAPIKEY }}
```

Comment:
This should be `NuGetApiKey`..
I have tested myself if this was the problem (apparently not :) )